include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -16,6 +16,7 @@
 #ifndef _ONEDPL_EXECUTION_SYCL_DEFS_H
 #define _ONEDPL_EXECUTION_SYCL_DEFS_H
 
+#include "../../../internal/common_config.h"
```

Comment:
It seems `common_config.h` is only included from public top level headers until this addition. Perhaps this is unnecessary now, as it is already in `include/oneapi/dpl/experimental/kernel_templates`

Reply:
I have removed it